docs/integration/index.md

```diff
@@ -0,0 +1,90 @@
+# DTS Integration Guide
+
+This document lists all of the things you must do in order to integrate your
+database with the BER Data Transfer System (DTS) in order to take advantage of
+its file and metadata transfer capabilities.
+
+We have tried to cover all the necessary topics comprehensively here, but
+there's no substitute for a real person when confusion arises, so please don't
+hesitate to contact the KBase DTS development team with your questions.
+Take a look at the [DTS Integration Glossary](glossary.md) for an explanation of
+the terminology used in this guide.
+
+The guidance we present here is not intended to be prescriptive. We provide
+suggestions and examples of technical components to illustrate how your
+organization can integrate a database with the DTS, but in actuality the DTS
+is very flexible and can accommodate various implementations. For example,
+we may be able to adapt existing capabilities for DTS integration in certain
+situations.
+
+## Overview
+
+The DTS provides a file transfer capability whose organizational unit is
+**individual files**. We're not in the business of telling researchers how to
```

Comment:
it may be worth saying something to cover databases that don't have the raw data as files -- it's all just in a database. In those cases, the system would just need to be able to output a file in whatever the appropriate format is.